src/guides/resources.md

```diff
@@ -0,0 +1,138 @@
+## Handshake Resource Guide
+
+Handshake has resource records that represent DNS resource records.
+Some of them map one-to-one with DNS resource record types,
+while others do not. These resource records are included in
+`REGISTER` or `UPDATE` outputs and written to the Urkel tree
+so that proofs can be served to light clients.
+
+### Resource Record Types
+
+- `DS`
+- `NS`
+- `GLUE4`
+- `GLUE6`
+- `SYNTH4`
+- `SYNTH6`
+- `TXT`
+
+#### `DS`
+
+The `DS` Handshake resource record is the same as the DNS
+`DS` resource record. It it used for DNSSEC and is specified
+in [RFC 4033](http://tools.ietf.org/html/rfc4033),
+[RFC 4034](http://tools.ietf.org/html/rfc4034) and
+[RFC 4035](http://tools.ietf.org/html/rfc4035).
+Its purpose it to enable secure delegation of zones by creating
+a chain of authentication between a parent and child zone.
+The `DS` record contains the hash of a `DNSKEY` resource record.
+It is inserted into the parent zone and the `DNSKEY` record
+is inserted into the child zone.
+The Handshake `DS` record must contain the fields `type`,
+`keyTag`, `algorithm`, `digestType` and `digest`.
+See [RFC 4034](https://tools.ietf.org/html/rfc4034) for the
+definitions of these fields.
+
+Example:
+
+```json
+{
+  "records": [{
+    "type": "DS",
+    "keyTag": 24620, 
+    "algorithm": 8,
+    "digestType": 2, 
+    "digest": "297595dc199b947aa8650923619436fbdfd99fd625195111ab4efe950900cade"
+  }]
+}
+```
+
+#### `NS`
+
+The `NS` Handshake resource record is the same as the DNS
+`NS` resource record. It is used to delegate authority of a name
+to an authoritative nameserver. It is specified in
+[RFC 1035](https://tools.ietf.org/html/rfc1035).
+The Handshake `NS` record must contain the fields `type` and `ns`.
+The `ns` field must be a fully qualified domain name, including
+a trailing dot. It is useful when the `ns` name is resolvable
+via ICANN.
+
+Example:
+
+```json
+{
+  "records": [{
+    "type": "NS",
+    "ns": "ns1.easyhandshake.org."
+  }]
+}
+```
+
+#### `GLUE4`
+
+The `GLUE4` Handshake resource record corresponds to two DNS 
+resource records, a `NS` record and an `A` record. The `A`
+record is used as the glue. `A` records are defined in
+[RFC 1035](https://tools.ietf.org/html/rfc1035) and must contain
+an IPv4 address. The Handshake `GLUE4` record must contain the
+fields `type`, `ns` and `address`. It is useful for setting
+up an authoritative nameserver to manage subdomains for
+a Handshake name.
```

Comment:
Wanna mention that glue A/AAAA records are inserted into the "additional" section? (maybe link another RFC?)